examples/dns/Cargo.toml

```diff
@@ -19,8 +19,6 @@ path = "lib.rs"
 crate-type = [
     # Used for normal contract Wasm blobs.
     "cdylib",
```

Comment:
We can simpliy this to `crate-type = "cdylib"` now.

Reply:
Or does this actually work if we still want to execute the contract crate via `cargo test`?

Reply:
`cargo test` still works

Reply:
> We can simpliy this to `crate-type = "cdylib"` now.

Arpparently it needs to be an array `invalid type: string "cdylib", expected a sequence for key `lib.crate-type``